### ISSUE TITLE
New groc install fails due to breaking changes in underscore 1.7

### ISSUE THREAD

**Issue by scottnonnenberg** [nevir/groc#167]
This is the error:

```
! Failed to compile /node_modules/groc/lib/styles/default/behavior.coffee: Object function (data) {
        return render.call(this, data, _);
      } has no method 'charCodeAt'
```

This is the core underscore.js issue: https://github.com/jashkenas/underscore/issues/1805

Full set of repeatable steps:

``` bash
$ npm install groc

groc@0.6.3 node_modules/groc
├── colors@0.6.2
├── spate@0.1.0
├── underscore@1.7.0
├── showdown@0.3.1
├── optimist@0.6.1 (wordwrap@0.0.2, minimist@0.0.10)
├── fs-tools@0.2.11 (async@0.2.10, lodash@2.4.1)
├── coffee-script@1.8.0 (mkdirp@0.3.5)
├── glob@4.0.5 (once@1.3.0, inherits@2.0.1, graceful-fs@3.0.2, minimatch@1.0.0)
├── uglify-js@2.4.15 (uglify-to-browserify@1.0.2, async@0.2.10, optimist@0.3.7, source-map@0.1.34)
└── jade@1.5.0 (character-parser@1.2.0, commander@2.1.0, constantinople@2.0.1, with@3.0.1, mkdirp@0.5.0, transformers@2.1.0, monocle@1.1.51)

$ ./node_modules/groc/bin/groc node_modules/groc/lib/**/*

  Generating documentation...
✓ /node_modules/groc/lib/styles/base.html
» /node_modules/groc/lib/styles/default is not in a supported language, skipping.
✓ /node_modules/groc/lib/styles/default.html
✓ /node_modules/groc/lib/utils/cli_helpers.html
✓ /node_modules/groc/lib/utils/compatibility_helpers.html
✓ /node_modules/groc/lib/utils/humanize.html
✓ /node_modules/groc/lib/utils/logger.html
✓ /node_modules/groc/lib/utils/style_helpers.html
! Failed to compile /node_modules/groc/lib/styles/default/behavior.coffee: Object function (data) {
        return render.call(this, data, _);
      } has no method 'charCodeAt'
```

**Comment by Schoonology** [comment]
Do we feel comfortable just updating the underscore version to `~1.6.0`? (Wound up doing that locally, would be happy to submit a PR.)

**Comment by make-github-pseudonymous-again** [comment]
+1

**Comment by stojanovic** [comment]
+1

**Comment by emuvente** [comment]
+1

**Comment by cef62** [comment]
+1

**Comment by OlenDavis** [comment]
+1

**Comment by penx** [comment]
Schoonology's suggestion works for me and makes sense, in the short term at least, so I've submitted a pull request

**Comment by deanrad** [comment]
+1

**Comment by nevir** [comment]
This should be fixed as of groc 0.7.0 - lemme know if you run into issues
